federation/planner.go

```diff
@@ -122,19 +123,19 @@ func printPlan(rootPlan *Plan) {
 	}
 }
 
-func printSelections(selectionSet *graphql.SelectionSet) {
```

Comment:
Thanks for cleaning up the `printSelections` logic! Should make debugging easier in the future!